cli/localci/cli/run.py

```
@@ -326,16 +380,91 @@ def run(
+        lines = f.readlines()
+
+    if image_tag:
+        name_escaped = re.escape(entry.name)
+        name_pattern = re.compile(r'name:\s*["\']?' + name_escaped + r'["\']?\s*$')
+        name_idx = None
+        for i, line in enumerate(lines):
+            if name_pattern.search(line.strip()):
+                name_idx = i
+                break
+        if name_idx is None:
+            raise ValueError(f"Matrix entry name '{entry.name}' not found in workflow")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)